### ISSUE TITLE
Report offset?

### ISSUE THREAD

**Issue by antoinefalisse** [stanfordnmbl/opencap-core#37]
We might want to report the offset we add to the marker data: https://github.com/stanfordnmbl/opencap-core/blob/main/main.py#L357 in case people want to sync OpenCap data with data from other devices. Thoughts @suhlrich ?

**Comment by suhlrich** [comment]
Agreed. That was my thought for responding to that forum post. Will think how to best do that.

On Mar 2, 2023, at 4:54 PM, Antoine Falisse ***@***.***> wrote:

﻿

We might want to report the offset we add to the marker data: https://github.com/stanfordnmbl/opencap-core/blob/main/main.py#L357 in case people want to sync OpenCap data with data from other devices. Thoughts @suhlrich<https://github.com/suhlrich> ?
